onebusaway-android/src/main/java/org/onebusaway/android/app/Application.java

```diff
@@ -70,6 +73,9 @@
     // Region preference (long id)
     private static final String TAG = "Application";
 
+    public static final String CHANNEL_REALTIME_ID = "ch101";
```

Comment:
Could you please change the name of `CHANNEL_REALTIME_ID` to `CHANNEL_TRIP_PLAN_UPDATES_ID`?